frontend/src/layouts/GitlabDashboard/GitlabApp.js

```diff
@@ -0,0 +1,140 @@
+import { React, useEffect, useState } from "react";
+// import "./App.css";
+import LoginButton from "../../components/GitlabButtons/LoginButton";
+import LogoutButton from "../../components/GitlabButtons/LogoutButton";
+import Profile from "./GitlabProfile";
+import { Table } from "baseui/table-semantic";
+import { useAuth0 } from "@auth0/auth0-react";
+import { Spinner } from "baseui/spinner";
+const axios = require("axios");
+
+function GitlabApp() {
+    const [loading, setLoading] = useState(false);
+    const [currentPage, setCurrentPage] = useState(1);
+    const [postsPerPage] = useState(4);
+    const [posts, setPosts] = useState([]);
+
+    useEffect(() => {
+        const fetchProjectids = async () => {
+            const userdata = await axios.get(
+                "https://gitlab.com/api/v4/user?access_token=a47babe4ca273839ce99dc4e4a568ec343a17b3ca830128699c32b0f4e1c5555"
+            );
+            //console.log(userdata);
+            const username = userdata.data.username;
+            //console.log(username);
+            const res = await axios.get(
```

Comment:
async await is one way to make sequential calls but do read on how to solve this use case using promises as well.